packages/pwa/src/components/FeedItemAdder.tsx

```
@@ -25,6 +25,11 @@ export const FeedItemAdder: React.FC = () => {
     <FlexColumn gap={12} align="flex-start">
       <input type="text" value={url} onChange={(e) => setUrl(e.target.value)} />
       <button onClick={() => handleAddItemToQueue(url)}>Add to import queue</button>
+      <button
+        onClick={() => handleAddItemToQueue('https://jwn.gr/posts/migrating-from-gatsby-to-astro/')}
+      >
```

Biome

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)